src/main/resources/db/migration/V202409231510__create_company_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE company
```

Comment:
best practice is to do this:
```suggestion
CREATE TABLE IF NOT EXISTS company
```